ext/libxml/libxml.c

```diff
@@ -1077,11 +1086,9 @@ PHP_FUNCTION(libxml_use_internal_errors)
 /* {{{ Retrieve last error from libxml */
 PHP_FUNCTION(libxml_get_last_error)
 {
-	xmlErrorPtr error;
-
 	ZEND_PARSE_PARAMETERS_NONE();
 
-	error = xmlGetLastError();
+	const xmlError* error = xmlGetLastError();
```

Comment:
This looks like a bugfix that probably should be backported? Or is this dependent on the libxml version?

Reply:
This change will work on old and new versions. This "only" causes a warning, which only causes issues on `-Werror` builds but we don't typically backport warnings right?

Reply:
We don't usually do yes, but the main thing is that I miss read what the type change is, as the fact it was changing made me thing we had a bug in the current implementation.

So this can be ignored.